src/game_system.cpp

```diff
@@ -580,6 +580,38 @@ void Game_System::OnSeReady(FileRequestResult* result, lcf::rpg::Sound se, bool
 			Output::Debug("Sound not found: {}", result->file);
 			return;
 		}
+
+		if (Player::IsPatchKeyPatch() && StringView(result->file).ends_with(".link") && stream.GetSize() < 500) {
+			// Handle Ineluki's MP3 patch
+			std::string line = InelukiReadLink(stream);
+
+			// Needs another Async roundtrip
+			FileRequestAsync* request = AsyncHandler::RequestFile(line);
+			se_request_ids[line] = request->Bind(&Game_System::OnSeInelukiReady, this, se);
+			request->Start();
+			return;
+		}
+
+		se_cache = AudioSeCache::Create(std::move(stream), result->file);
+	}
+
+	if (!se_cache) {
+		Output::Warning("Sound {}: Format not supported", result->file);
```

Comment:
Maybe this brings warnings where RPG_RT.exe don't? Use Output::Debug instead?

Reply:
this is copy-pasted from the non-Ineluki SE handler.
The warning will only trigger when a SE file is unsupported format wise. Missing files won't trigger it so should be save imo